server/src/main.py

```diff
@@ -50,6 +52,9 @@ async def show_calendar():
 @app.post("/files", status_code=200)
 async def get_deadlines(response: Response, outline_file: UploadFile = File(...)):
     """Returns the extracted dates and info from the uploaded file"""
+    file_size = os.fstat(outline_file.file.fileno()).st_size
+    if file_size > 150000:
```

Comment:
My only suggestion would be to change this to a bit bigger number (I have a few outlines over 250 kb). Maybe 400kb and probably set it as a constant. So something like `MAX_FILE_SIZE = 400000` after the import statements and then use it here